.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  'env': {
+    'browser': true,
+    'es2021': true,
+  },
+  'extends': [
+    'google',
+  ],
+  'parser': '@typescript-eslint/parser',
+  'parserOptions': {
+    'ecmaVersion': 'latest',
+    'sourceType': 'module',
+  },
+  'plugins': [
+    '@typescript-eslint',
+  ],
+  'rules': {
```

Comment:
You need to add the rules that are on the guide repository

Reply:
[Análisis de código estatico](https://github.com/holgiosalos/cypress-training-doc#6-an%C3%A1lisis-de-c%C3%B3digo-estatico)